### PR TITLE
ios: Update LinkKit to 1.0.6

### PR DESCRIPTION
**Pull request by afh**

### CODE REVIEW

**Comment by steckel** [comment]
LGTM :+1:

File: ios/CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # CHANGELOG
 
+## 2017-07-11 — LinkKit 1.0.6
+### Added
+- Add [copy customization](https://blog.plaid.com/link-copy-customization/) which allows to change the text of certain user interface elements in the Link flow
```
Comment (pbernasconi) [inline ios/CHANGELOG.md]: nit: typo in `allows to change` – using copy from the blog why not say: `which gives you the ability to customize copy in the Link flow`.

File: ios/CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+- Add exit button when searching for an institution yielded no results so people can directly exit out of Link iOS
```
Comment (pbernasconi) [inline ios/CHANGELOG.md]: What do you think about using copy from the plaid.com/docs/api#onexit-callback :

```
- Add an exit button, to handle the situation when a user exited the Link flow after unsuccessfully (no results returned) searching for a financial institution
```

File: ios/LinkDemo-ObjC/LinkDemo-ObjC/ViewController.m [hunk 3]
```diff
@@ -63,6 +63,7 @@ - (void)didReceiveNotification:(NSNotification*)notification {
 - (IBAction)didTapButton:(id)sender {
 #if USE_CUSTOM_CONFIG
     [self presentPlaidLinkWithCustomConfiguration];
+    // [self presentPlaidLinkWithCopyCustomization];
```
Comment (pbernasconi) [inline ios/LinkDemo-ObjC/LinkDemo-ObjC/ViewController.m]: should we remove this?

File: ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift [hunk 4]
```diff
@@ -48,6 +48,7 @@ class ViewController: UIViewController {
     @IBAction func didTapButton(_ sender: Any?) {
 #if USE_CUSTOM_CONFIG
         presentPlaidLinkWithCustomConfiguration()
+        // presentPlaidLinkWithCopyCustomization()
```
Comment (pbernasconi) [inline ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift]: can we remove this?

File: ios/LinkKit.framework/Headers/PLKConfiguration.h [hunk 5]
```diff
@@ -46,6 +46,41 @@ PLK_EXTERN NSString* const kPLKTestKey;
...
+/// The text shown as the navigation bar title.
+PLK_EXTERN NSString* const kPLKCustomizationTitleKey;
+
+/// The text shown upon successful (re)connection of an account.
+PLK_EXTERN NSString* const kPLKCustomizationMessageKey;
+
+/// The text shown on the submit button.
+PLK_EXTERN NSString* const kPLKCustomizationSubmitButtonKey;
+
+/// The text shown on the button on the bottom of the institution select view to ?.
```
Comment (pbernasconi) [inline ios/LinkKit.framework/Headers/PLKConfiguration.h]: is the question mark here on purpose?

File: ios/LinkDemo-ObjC/LinkDemo-ObjC/ViewController.m (hunk 3 above)
Comment (afh) [inline ios/LinkDemo-ObjC/LinkDemo-ObjC/ViewController.m]: No, this way people can easily try out instance configuration based customization.

File: ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift (hunk 4 above)
Comment (afh) [inline ios/LinkDemo-Swift/LinkDemo-Swift/ViewController.swift]: No, this way people can easily try out instance configuration based customization.

File: ios/LinkKit.framework/Headers/PLKConfiguration.h (hunk 5 above)
Comment (afh) [inline ios/LinkKit.framework/Headers/PLKConfiguration.h]: It is not.

**Pull request merged**